packages/plugin-core/src/FeatureShowcase.ts

```diff
@@ -0,0 +1,78 @@
+import { VSCodeEvents } from "@dendronhq/common-all";
+import { SegmentClient } from "@dendronhq/common-server";
+import { MetadataService, ShowcaseEntry } from "@dendronhq/engine-server";
+import * as vscode from "vscode";
+import {
+  MeetingNoteTestGroups,
+  MEETING_NOTE_FEATURE_SHOWCASE_TEST,
+} from "./abTests";
+import { AnalyticsUtils } from "./utils/analytics";
+
+/**
+ * Class to showcase certain features of Dendron (like a tip of the day). Right
+ * now, we have only 1 feature - we can expand functionality in this class as we
+ * have more tips.
+ */
+export class FeatureShowcase {
+  /**
+   * Show right now uses vscode.showInformationMessage.  Later, we can have
+   * different display locations, such as the proposed Dendron Side Panel
+   */
+  public show(): void {
+    const ABUserGroup = MEETING_NOTE_FEATURE_SHOWCASE_TEST.getUserGroup(
+      SegmentClient.instance().anonymousId
+    );
+
+    if (
+      ABUserGroup === MeetingNoteTestGroups.show &&
+      MetadataService.instance().getFeatureShowcaseStatus(
+        ShowcaseEntry.TryMeetingNotes
+      ) === undefined
+    ) {
+      this.showMeetingNotesTip();
+    }
+  }
+
+  /**
+   * Tip to try out meeting notes
+   */
+  private showMeetingNotesTip() {
+    const confirm = "Try It";
+    const reject = "Later";
+
+    vscode.window
+      .showInformationMessage(
+        `Would you like to try out the new Meeting Notes Feature?`,
```

Comment:
would change the text to provide more context. 

Dendron now has meeting notes which makes it easy to keep track of your meetings. Would you like to try it out?

"Create a meeting note", "Later"

Reply:
One consideration here though is that I _think_ it's a little bit better if the text fits in one line, otherwise the user needs to click another button before they can see the entire text and may lose interest.

Reply:
I'm inclined to shorten it to "Dendron now has meeting notes - Try it out!"
[ Create a Meeting Note ] [Later]

We can A/B test it later.